docs/topics/api/categories.rst

```diff
@@ -0,0 +1,125 @@
+==========
+Categories
+==========
+
+.. note::
+
+    These APIs are experimental and are currently being worked on. Endpoints
+    may change without warning.
+
+.. _category-list:
+
+Categories are defined by a name, a slug, a type and an application. Slugs are
+only guaranteed to be unique for a given ``app`` and ``type`` combination, and
```

Comment:
Describing `slug` as being _namespaced_ by `app` and `type` makes more sense to me than saying how its uniqueness is scoped, but if you disagree ignore me 😄